src/services/config-service/entities/chain.entity.ts

```diff
@@ -1,4 +1,4 @@
-export interface SafeConfigChain {
```

Comment:
Both `Page` and `Chain` have exact duplicates under `src/chains/entities`. I was thinking about we could use those interfaces directly... from a DDD/SOLID perspective I think it's not incorrect to make use of _domain_ classes into this kind of _adapters_ layer. And if we found any use case which forces us to have differences between them, we could create a DTO in the _adapters_ layer.

Reply:
For `Page`, if it's a common interface, maybe we could extract it to a `src/shared` folder (and remove it from `src/chains`), what do you think?

Reply:
I agree. In this case however `src/chains` was created initially as a demo. Its `Page` should actually be more like the following: https://github.com/safe-global/safe-client-gateway/blob/11ce5e74a8bd670e91fe547e37eb71afe5b835b6/src/common/models/page.rs#L5-L12

The concept of `Page` would actually need to be used across every feature. For this I'd create an entity under a new folder (like `common`).

If `Chain` ends up being the same object then we can move it to this `/common` folder that it can be used by both the `ConfigService` and the features.